.gitmodules

```diff
@@ -246,6 +246,7 @@
 	url = https://github.com/TexasInstruments/cc13xx_cc26xx_sdk.git
 	branch = main
 	platforms = cc13x2_26x2
+	ignore = dirty
```

Comment:
Why is this required?

Reply:
Using the python script now drops a `__pycache__` in the tools directory. I didn't notice it when creating the repo because I had used the compiled executable. I could probably update the repository's .gitignore too.

Reply:
how about adding to the top of the file a `#!/usr/bin/env -S python3 -B` and that should avoid creating of cache files.

our tools/builds should not dirty the source tree.